admin.go

```diff
@@ -109,6 +109,12 @@ type ConfigSettings struct {
 	//
 	// EXPERIMENTAL: Subject to change.
 	LoadRaw json.RawMessage `json:"load,omitempty" caddy:"namespace=caddy.config_loaders inline_key=module"`
+
+	// The interval to pull config. With a non-zero value, will pull config
+	// from config loader (eg. a http loader) with given interval.
+	//
+	// EXPERIMENTAL: Subject to change.
+	PullInterval caddy.Duration `json:"pull_interval,omitempty"`
```

Comment:
This should be just `Duration` because we're in the `caddy` package already in here

```suggestion
	PullInterval Duration `json:"pull_interval,omitempty"`
```